website/content/api-docs/system/storage/raft.mdx

```diff
@@ -52,6 +52,11 @@ leader node.
 
 - `auto_join_port` `(int: 8200)` - Port to be used for `auto_join`.
 
+- `non_voter` `(bool: false)` - If set, will make the server not
+  participate in the Raft quorum, and have it only receive the data replication
+  stream. This can be used to add read scalability to a cluster in cases where a
+  high volume of reads to servers are needed. The default is false.
```

Comment:
One nit if you want to look it up: does adding non-voter nodes increase application latency? Is data  sent to non-voter nodes handled inline to the application or is is queued separately? If you run a cluster with, say, 5 voters and 206 non-voters, does this greatly increase the time to apply logs still?

Just something we might want to call out in docs if it does.

Reply:
Honestly I am not sure. I copy pasted the text from the previous enterprise docs

Reply:
yeah, perf will take a hit as leader will be replicating data to all those non-voters. I would throw in a caution on fine tuning this for intended use case.

Reply:
I am not familiar with raft library as such so can't estimate this easily. Conceptually, makes sense but don't know how hard it is. need a spike.